.github/workflows/test.yml

```diff
@@ -30,21 +30,33 @@ jobs:
     - name: Run Memgraph MAGE image
       run: |
         docker run -d -p 7687:7687 memgraph:mage --telemetry-enabled=False
+    - name: Set up C++
+      run: |
+        sudo apt update
+        sudo apt install -y build-essential cmake
     - name: Set up Python
       uses: actions/setup-python@v2
       with:
         python-version: '3.7'
-    - name: Install dependencies
-      run:
+    - name: Build C++ modules
+      run: |
+        mkdir -p cpp/build
+        (cd cpp/build && cmake .. && make -j8)
```

Comment:
Why do you put everything in one row if you can change them just by calling the cmds row by row

Reply:
I propose making this efficient in the separate ease of use PR, and wait till that one is solved?

Reply:
For sure!